docs/source/status.rst

```diff
@@ -72,11 +72,11 @@ Data Types
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
 | Large Utf8        | ✓     | ✓     | ✓     | ✓  | \(4)  |  ✓    | ✓     |       | ✓         |
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
-| Binary View       | ✓     |       | ✓     |    |   ✓   |  ✓    |       |       |           |
+| Binary View       | ✓     |       | ✓     |    |   ✓   |  ✓    |       |       | ✓         |
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
 | Large Binary View | ✓     |       | ✓     |    |       |       |       |       |           |
```

Comment:
Does Large Binary View / Large Utf8 View exist? I couldn't see mention of this anywhere else in the codebase or protobuf spec